pkg/network/multus_admission_controller.go

```diff
@@ -82,16 +82,31 @@ func renderMultusAdmissonControllerConfig(manifestDir string, externalControlPla
 		data.Data["TokenAudience"] = os.Getenv("TOKEN_AUDIENCE")
 		data.Data["RunAsUser"] = hsc.RunAsUser
 
-		// Get serving CA from the management cluster since the service resides there
+		CAConfigMap, ok := os.LookupEnv("CA_CONFIG_MAP")
```

Comment:
To avoid duplicating this logic it should be moved to `pkg/hypershift/hypershift.go` similar to other HyperShift env vars.

Reply:
Ok moved this to hypershift.go